src/main/webapp/util/Validator.js

```diff
@@ -135,6 +136,34 @@ class Validator {
     }
 
     doValidation(data) {
+        if (this.isName) {
```

Comment:
have we removed duplicate validation from `handleFormSubmit` method?

Reply:
I did not moved that validation inside. as it was not required.